docs/internal_api.yml

```diff
@@ -86,6 +86,25 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
+  /tenants/{tenantId}:
+    delete:
+      tags:
+        - Internal API
+      operationId: Delete tenant
+      summary: Delete all data belonging to a given tenant.
+      parameters:
+        - in: path
+          name: tenantId
+          schema:
```

Comment:
do we need "schema" here?

Reply:
this is how this whole file is made, do you want me to change everywhere?